packages/vite/src/client/client.ts

```diff
@@ -333,6 +333,10 @@ async function waitForSuccessfulPing(
 }
 
 const sheetsMap = new Map<string, HTMLStyleElement>()
+document.querySelectorAll('style[data-vite-dev-id]').forEach((el) => {
+  sheetsMap.set(el.getAttribute('data-vite-dev-id')!, el as HTMLStyleElement)
+})
```

Comment:
I think it would be good to add a comment here to help others understand this part. So we avoid the risk of someone deleting it as unneeded in the future

Reply:
Done!